ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManagerMetrics.java

```diff
@@ -58,6 +58,8 @@ class HelixClusterManagerMetrics {
   public Gauge<Long> helixClusterManagerCurrentXid;
   public final Timer routingTableQueryTime;
   public final Counter resourceNameMismatchCount;
+  // Number of times we chose a local partition that was not fully available remotely for a put operation.
+  public final Counter ineligibleReplicaCount;
```

Comment:
nit: Lets try to use a better name.`ineligibleReplicaCount` doesn't reflect the actual purpose of this metric. A variable name that would make the comment in the line above redundant would be the most appropriate. Maybe `paranoidDurabilityIneligibleReplica`?

Reply:
Yep, changed the variable name.